packages/bot-skeleton/src/scratch/dbot.js

```diff
@@ -231,6 +231,7 @@ class DBot {
         api_base.setIsRunning(false);
         if (this.interpreter) {
             this.interpreter.stop();
+            this.interpreter.terminateSession();
```

Comment:
@farabi-deriv did not follow this, why do we need it. I thought terminateSession is called inside stop

<img width="704" alt="image" src="https://github.com/binary-com/deriv-app/assets/100689171/6328ed0e-4e8e-4e80-b5ec-32260f0bf076">

Reply:
@vinu-deriv we need to terminate the bot immediately on stopping the bot. At the moment if a contract has 15 minutes duration, we are unable to trade again until the duration is complete. So i added the terminateSession(). We could actually remove the else if condition aswell as we no longer need to check for contract status to be sold first. It should be running on the reports page.